packages/xdl/src/Simulator.js

```diff
@@ -165,6 +193,21 @@ async function _waitForSimulatorRunningAsync() {
   }
 }
 
+async function _getFirstAvailableDeviceAsync() {
+  let simulatorDeviceInfo = await _listSimulatorDevicesAsync();
+  for (let runtime in simulatorDeviceInfo.devices) {
+    let devices = simulatorDeviceInfo.devices[runtime];
+    for (let i = 0; i < devices.length; i++) {
+      let device = devices[i];
+      if (device.isAvailable && device.name.includes('iPhone')) {
+        return device;
+      }
+    }
+  }
+  console.warn('No iPhone devices available in Simulator.')
```

Comment:
```suggestion
  Logger.global.warn('No iPhone devices available in Simulator.')
```
(All logging in XDL goes through `Logger`. This also makes it work in Expo Dev Tools and not only in terminal.)